modules/ROOT/pages/functions/mathematical-logarithmic.adoc

```diff
@@ -217,7 +219,8 @@ log10(expression)
 |===
 
 | `log10(null)` returns `null`.
-| `log10(0)` returns `null`.
+| `log10(0)` returns `-Infinity`.
+| If (`expression` < 1), then (`log10(expression)`) returns `NaN`.
```

Comment:
```suggestion
| If (`expression` < 0), then (`log10(expression)`) returns `NaN`.
```

Reply:
We might want to rewrite sqrt in the same way to align, right now it says

sqrt(<any negative number>) returns NaN